src/it/resources/org/checkstyle/suppressionxpathfilter/descendanttoken/InputXpathDescendantTokenSwitchTooManyCases.java

```diff
@@ -0,0 +1,16 @@
+package org.checkstyle.suppressionxpathfilter.descendanttoken;
+
+public class InputXpathDescendantTokenSwitchTooManyCases {
+
+    void testMethod1() {
+        int x = 1;
+        switch (x) { // warn
```

Comment:
Please put this switch into lambda

Reply:
Please move input files to non-compiled folder, in such case jdk 11 will not be used to compile it.

Reply:
@Mostafahassen1 , please do this update